packages/react/ds-core-form/src/ui/Field/inputs/Textarea/Textarea.t-ests.tsx

```diff
@@ -0,0 +1,21 @@
+/* @canonical/generator-ds 0.9.0-experimental.4 */
+
+import { render, screen } from "@testing-library/react";
+import * as decorators from "storybook/decorators.js";
+import { beforeEach, describe, expect, it } from "vitest";
+import Component from "./Textarea.js";
+
+describe("Textarea component", () => {
+	// beforeEach((context) => {
+	//   context.test = decorators.form()(context.test);
+	// });
+	it("renders", () => {
+		render(<Component name="content">Textarea</Component>);
+		expect(screen.getByText("Textarea")).toBeInTheDocument();
```

Comment:
This is currently causing a failing build. I think this is because you have renamed your test files to `t-ests.tsx`, which is not excluded by `tsconfig.build.json`. If you intend to push these, I'd suggest either getting the tests to a place where they pass by functionality or bypassing the tests in a more standardized/declarative way, like this:

```tsx
describe("Textarea component", () => {
  it("test bypass", () => {
    expect(true).toBe(true);
  });
});

You will also need to rename the files back to `*.tests.tsx` so they are excluded by the build tsconfig.

This should allow your tests to "pass" while also avoiding a build failure.